packages/internals/src/WasmPanicRegistry.ts

```diff
@@ -0,0 +1,12 @@
+export class WasmPanicRegistry {
+  #message = ''
```

Comment:
It may not be currently safe to use private fields, depending on whether they end up in typings: https://prisma-company.slack.com/archives/CUXLS0Z6K/p1675164034311419. Could be okay here though. Better to run ecosystem-tests before merging to make sure.

Reply:
I've simply removed the `#` prefix and replaced it with `private`.